src/core/client/stream/components/Permalink/Permalink.tsx

```diff
@@ -0,0 +1,38 @@
+import { Localized } from "fluent-react/compat";
+import React from "react";
+import { Button, ButtonIcon, Popover } from "talk-ui/components";
+import PermalinkPopover from "./PermalinkPopover";
+
+interface InnerProps {
```

Comment:
Call it `PermalinkProps`? We usually used `InnerProps` when we use HOCs but we don't here.